src/Application/Redux/actions.js

```diff
@@ -0,0 +1,34 @@
+export const CLICK_MONEY = 'CLICK_MONEY';
+export const COLLECT_INCOME = 'COLLECT_INCOME';
+export const BUY_ASSET = 'BUY_ASSET';
+export const UNLOCK_ASSET = 'UNLOCK_ASSET';
+export const UPGRADE_CURRENCY = 'UPGRADE_CURRENCY';
+export const BROADCAST_NEWS = 'BROADCAST_NEWS';
```

Comment:
we tend never to mix named exports with default export, i think we ran into it breaking... something... but that was a while ago and things might ahve gotten better.  also we'd just put these in a `ActionType.js` file that returns `Object.freeze({these fields...})`